src/myuw-notifications.js

```diff
@@ -188,6 +190,21 @@ export class MyUWNotifications extends HTMLElement {
       }
     }
 
+    /**
+     * Close menu if current focused (tabbed) element is outside the component
+     */
+    autoCloseMenu(event) {
+      this.focusableElements = !this.focusableElements ? this.getFocusableElements() : this.focusableElements;
```

Comment:
Could we do without `this.focusableElements` entirely? How bad would it be to just call `this.getFocusableElements()` whenever needed?

Reply:
I agree that there is a room for general improvement in the readability of focusNext() focusPrevious() and autoCloseMenu(). I'll help you look into it, Kesong!